torch_xla_py/data_parallel.py

```diff
@@ -15,6 +15,19 @@
 import traceback
 
 
+class Context(object):
+
+  def __init__(self, device):
+    self.device = device
```

Comment:
Can you help me understand why device is needed here?

Reply:
Just a convenience if someone passes the Context around, to have the device it refers to, available.

Reply:
Didn't see that it'd be used anywhere since `context` object is used within loop_fn and we already have `device` in that scope. But yes, that makes sense for debugging purposes and passing around.